IngresseSDK/Classes/IngresseService.swift

```diff
@@ -91,7 +91,7 @@ public class IngresseService {
         }
     }
     
-    public func restPOST(_ url: String, parameters: [String : AnyObject], handler: @escaping (_ success: Bool, _ response: [String:Any]) -> ()) {
+    open func restPOST(_ url: String, parameters: [String : AnyObject], handler: @escaping (_ success: Bool, _ response: [String:Any]) -> ()) {
```

Comment:
change the method names from `rest*` to just the action like: `GET` or `POST`.
Ex:
```
open func POST(...)
open func GET(...)
...
```